docs/preview/features/powershell/azure-logic-apps.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Scripts related to interacting with Azure Logic Apps"
+layout: default
+---
+
+# Azure Logic Apps
+
+This module provides the following capabilities:
+- [Disabling Azure Logic Apps from configuration file](#disabling-azure-logic-apps-from-configuration-file)
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.LogicApps
+```
+
+## Disabling Azure Logic Apps from configuration file
+
+Typically done the first task of the release pipeline, right before the deployment of the Logic Apps, will disable all specified Logic Apps in a specific order. 
+The Logic Apps to be disabled and the order in which this will be done, will be defined in the passed-along configuration file.
```

Comment:
```suggestion
The Azure Logic Apps to be disabled and the order in which this will be done, will be defined in the provided configuration file.
The order of the Azure Logic Apps in the configuration file (bottom to top) defines the order in which they will be disabled by the script. The counterpart of this script used to enable the Azure Logic Apps, will take the order as specified (top to bottom) in the file.
```